packages/notifi-react-card/lib/components/subscription/EventTypeHealthCheckRow.tsx

```diff
@@ -62,8 +63,10 @@ export const EventTypeHealthCheckRow: React.FC<
 
   const isValueType = config && config.checkRatios.type === 'value';
   let ratios: CheckRatio[] = [];
+  let thresholdDirection: ThresholdDirection = 'below';
   if (isValueType) {
     ratios = config.checkRatios.value;
+    thresholdDirection = ratios[0].type;
```

Comment:
bigger than the scope of this ticket, but we should probably put "direction" on the HealthCheckEventType, instead of inside a CheckRatio